package.json

```diff
@@ -40,12 +44,13 @@
     }
   },
   "dependencies": {
+    "autoprefixer-loader": "^2.0.0",
     "autoprefixer-core": "^5.2.0",
     "babel": "^5.8.21",
     "babel-loader": "^5.3.2",
     "babel-plugin-typecheck": "^1.2.0",
     "babel-runtime": "^5.8.20",
-    "basscss-sass": "^1.0.2",
+    "basscss-sass": "nicolasiensen/basscss-sass#v1.0.2-branch",
```

Comment:
The problem is that the latest version of basscss is 6.x and we implemented
the front-end using version 1.x. So if we change it to latest it's
guaranteed that the front-end will break :(
On Feb 22, 2016 18:40, "Leticia F." <notifications@github.com> wrote:

> In package.json
> <https://github.com/ourcities/hub-client/pull/1#discussion_r53661599>:
>
> >      "autoprefixer-core": "^5.2.0",
> >      "babel": "^5.8.21",
> >      "babel-loader": "^5.3.2",
> >      "babel-plugin-typecheck": "^1.2.0",
> >      "babel-runtime": "^5.8.20",
> > -    "basscss-sass": "^1.0.2",
> > +    "basscss-sass": "nicolasiensen/basscss-sass#v1.0.2-branch",
>
> Should we use latest version? It seems that annoying bug is fixed: basscss/basscss-sass#6
> (comment)
> <https://github.com/basscss/basscss-sass/issues/6#issuecomment-183780817>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ourcities/hub-client/pull/1/files#r53661599>.
>